frontend/views/pages/BypassUI.vue

```diff
@@ -119,6 +119,14 @@ export default {
           finalize: () => {
             this.$router.push({ path: '/dashboard' }) // eslint-disable-line
           }
+        },
```

Comment:
Boyscoutting: noticing that on line 115 a callback function is being passed in that is no longer being used. Callback should be removed.